### ISSUE TITLE
On and off setting language and visual queue

### ISSUE THREAD

**Issue by paaljoachim** [easy-updates-manager/easy-updates-manager#202]
Hey

In regarding the language used: https://easyupdatesmanager.com/updates/

![screen shot 2018-01-09 at 08 03 07](https://user-images.githubusercontent.com/5323259/34708547-9479c896-f513-11e7-8b8b-e2c62fe46a73.png)

It says disallow updates.

The question is are updates for theme on or off? So clicking it makes the specific theme or themes disallow updates.

Doing some thinking out loud...

Updates on
Updates off

What about something visual an icon perhaps to show red to show that a plugin or theme etc update is off and in green for on? This would make it very clear what is on and what is off.

I notice the blue line along the left edge. This can be used and perhaps it is as a signal that both themes updates are on (it could be green). Beside a theme that is not being updated it could perhaps be red.

Bottom line is having words and a visual queue when something is to be updated and when something is not to be updated.

**Comment by paaljoachim** [comment]
(I am transferring my last post at New EUM Admin Interface issue here. Except the error issue going from General tab to another tab and then back again.)

The new buttons look really good.
![screen shot 2018-01-18 at 11 58 35](https://user-images.githubusercontent.com/5323259/35094876-f0f9f084-fc46-11e7-8e8d-1ed1c16733b1.png)

I am now looking at the Plugins and Themes tabs.
Example Theme Update Options as it is right now:
![screen shot 2018-01-18 at 11 41 59](https://user-images.githubusercontent.com/5323259/35094917-15fbad14-fc47-11e7-8962-d73982a431e7.png)

Plugins Update Options:
![screen shot 2018-01-18 at 12 00 13](https://user-images.githubusercontent.com/5323259/35094942-29ff67ec-fc47-11e7-8d6a-8fe02f790c53.png)

Suggestion for change:
An approach that uses the same method as in the new General screen:
(As one decides on going with an approach it is best to stick with it for various screens.)
![easy-updates-manager-plugins-disable-enable](https://user-images.githubusercontent.com/5323259/35095448-c97d9676-fc48-11e7-829f-b6539fbaeedf.jpg)

Plugins and Themes tabs:
- Using buttons that uses the same method as in General screen. Disable/Enable buttons.

**Comment by paaljoachim** [comment]
An alternative:
![easy-updates-manager-ui-on-off](https://user-images.githubusercontent.com/5323259/35152649-85bacbee-fd23-11e7-939a-a343d115ec29.jpg)

Click the manual update link to update it right away similar to standard way of updating.
Click the on/off switch for automatic updating.
The approach is a little different then the buttons in the General tab though...

**Comment by ronalfy** [comment]
Hey @paaljoachim I think I'm going to go with this approach. It's a blend of your and my thoughts.

![screen shot 2018-01-19 at 12 42 27 pm](https://user-images.githubusercontent.com/636521/35166016-0ad8db4e-fd16-11e7-8534-9023895ff7f6.png)

**Comment by paaljoachim** [comment]
Can you do a short writeup of the logic behind Plugin Updates and the Automatic Updates?

**Comment by ronalfy** [comment]
I created a video: https://www.youtube.com/watch?v=6g-XPxJ-bJM&feature=youtu.be

When you turn on plugin/theme updates, you can configure automatic updates for that plugin or theme.

When you turn off plugin/theme updates, automatic updates will not work, so we hide them.

**Comment by paaljoachim** [comment]
The video is not available...

**Comment by ronalfy** [comment]
Sorry I made the video private instead of unlisted. You should be able to
view it now.

**Comment by paaljoachim** [comment]
I saw the video.
I was still not able to grasp the idea of two different sets of on/off options...probably because it is morning and my head is not there yet...
I am still confused. It would be better to on is on.... and off is off. Independent of whatever other choices one should be able to click an on or off button. To update or not to update. Automatic or not. From the screenshot you shared with me and the latest dev version I noticed (thankfully) that there was just one set of on and off buttons. As the logic behind should make choices real simple for the user to understand when something is on or off. Just show exactly what is needed and nothing more to the user. Make it as simple as possible for the user to turn options on and off.

**Comment by ronalfy** [comment]
Just keep in mind that plugin updates and automatic updates are two
different things.

Let’s schedule a hangout because I really need you on board with this.

**Comment by paaljoachim** [comment]
**Options in Easy Updates Manager:**
Disable All Updates: should then also remove the option to update themes/plugins. As in no buttons are seen in pages/tabs screen next to each plugin/theme.
NB! Actually it should hide the Plugins/Themes etc tabs because it is disabled. It should also hide most options seen in General. To just show the master switch is turned to off. No need to show any other option. 

All Updates are turned on and
Automatic Updates default.
On - Off buttons are seen in the themes/plugins tabs/screens.

Automatic Updates On
On - Off buttons are seen in the themes/plugins tabs. Themes/plugins should by default be to On. IF one decides to turn off a plugin/theme then the Automatic Updates that were On goes into Custom mode.

Automatic Updates Off.
On - Off buttons are seen in the themes/plugins tabs. Themes/plugins should by default be to Off.
IF one decides to turn on a plugin/theme then the Automatic Updates that were Off goes into Custom mode.

From video:
Themes screen: Disable theme updates then automatic updates can not be configured. Can not be turned on or off. Turning theme update on then one can also configure Automatic Updates on-off. This is too complex. Makes it difficult to understand. That decision should not be done directly in the plugins/themes pages but in General under Automatic Updates area, as I described it.

**Comment by ronalfy** [comment]
@mpsparrow if you can weigh in on this, let me know. If we can't go into an agreement, I'll revert back to using text links.

**Comment by ronalfy** [comment]
After speaking with @mpsparrow we're going to go with the toggle approach. Thanks @paaljoachim for weighing in. We're under a tight deadline and can't discuss this further.

**Comment by paaljoachim** [comment]
Ok. That's fine. Happy to be helping out..:)
